### PR TITLE
Fix "Cannot read property 'dataset' of null"

### PR DESCRIPTION
**Pull request by dkniffin**
@jrattray13 I've got a fix for #47 & #49

### CODE REVIEW

**Comment by jrattray13** [comment]
This fixes the crash, but makes the colouring logic less reliable (Brackets change color based on/when scrolling) I will need to look into it more ..

File: lib/swackets-view.coffee [hunk 1]
```diff
@@ -107,7 +107,7 @@ class SwacketsView
         sortedLineGroups = lineGroups.sort (a, b) =>
             Math.min(1, Math.max(-1, b.style.zIndex - a.style.zIndex))
 
-        firstLine = sortedLineGroups[0].querySelector('.line')
+        firstLine = sortedLineGroups[0]
```
Comment (angeldeejay) [inline lib/swackets-view.coffee]: ```firstLine = if sortedLineGroups[0].classList.contains('line') then sortedLineGroups[0] else sortedLineGroups[0].querySelector('.line')```

Legacy approach

Comment (jrattray13) [reply lib/swackets-view.coffee]: I still see same issues with bracket colours changing / not matching after scrolling, are you not seeing this?

Comment (Liath) [reply lib/swackets-view.coffee]: My brackets have always changed color when scrolling. I actually kind of like it that way, disco mode.

**Comment by dkniffin** [comment]
@jrattray13 I'm wondering if it makes sense to merge this, so swackets (and atom) isn't completely broken, and then fix the but with the changing colors.

**Comment by jrattray13** [comment]
ALIAS I had some spare time! Swackets is not intuative unfortunately... but 1 evening later, now work again swacket-smooth for me on atom v0.19.2 let me know if it still is broke :)

**Pull request closed without merging**